pkg/neorpc/errors.go

```diff
@@ -27,36 +27,231 @@ const (
 )
 
 // RPC error codes defined by the Neo JSON-RPC specification extension.
+// Codes for missing items.
 const (
-	// RPCErrorCode is returned on RPC request processing error.
-	RPCErrorCode = -100
```

Comment:
You have to remove this constant usages in the same commit. Currently the node on this branch can't be built and tested:
```
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ git status
interactive rebase in progress; onto 11c0f13d
Last commands done (2 commands done):
   pick c7d666a6 core: split ErrAlreadyExists into ErrAlreadyExists and ErrAlreadyInPool
   edit 7db25981 neorpc: add error codes and response errors
Next commands to do (5 remaining commands):
   pick 00acf119 rpcsrv: remove default request HTTP codes from switch statement
   pick 1527b56b rpcsrv: return error on unknown storage item from `getstorage`
  (use "git rebase --edit-todo" to view and edit)
You are currently editing a commit during a rebase.
  (use "git commit --amend" to amend the current commit)
  (use "git rebase --continue" once you are satisfied with your changes)

Untracked files:
  (use "git add <file>..." to include in what will be committed)

	n3testnet.json
	t_test.go

nothing added to commit but untracked files present (use "git add" to track)
anna@kiwi:~/Documents/GitProjects/nspcc-dev/neo-go$ make build
=> Building binary
+ export GOGC=off
+ export CGO_ENABLED=0
+ go build -trimpath -v -ldflags -X 'github.com/nspcc-dev/neo-go/pkg/config.Version=0.101.5-pre-216-g7db25981' -X 'github.com/nspcc-dev/neo-go/cli/smartcontract.ModVersion=v0.0.0-20230810094130-11bb733f1a2c' -o ./bin/neo-go ./cli/main.go
github.com/nspcc-dev/neo-go/pkg/services/rpcsrv
# github.com/nspcc-dev/neo-go/pkg/services/rpcsrv
pkg/services/rpcsrv/error.go:47:41: undefined: neorpc.RPCErrorCode
Makefile:31: recipe for target 'build' failed
make: *** [build] Error 2
```

And what we need is that at every commit the node should be successfully built and tested. It is needed for the bisect search over the git history in case of bugs investigation.